envoy/access_log/access_log.h

```diff
@@ -104,5 +104,8 @@ using FilterPtr = std::unique_ptr<Filter>;
 using Instance = InstanceBase<Formatter::HttpFormatterContext>;
 using InstanceSharedPtr = std::shared_ptr<Instance>;
 
+using InstanceInlinedVector = absl::InlinedVector<InstanceSharedPtr, 4>;
+using InstanceVector = std::vector<InstanceSharedPtr>;
```

Comment:
nit: consider keeping the explicit uses of `std::vector<AccessLog::InstanceSharedPtr>` or change the name to reflect that this is a vector of shared pointers (and not of Instance types).